sample/Java/HttpTrigger/Function.java.txt

```diff
@@ -1,3 +1,6 @@
+// This is a .txt extension intentionally so CodeQL does not expect and wait for a java compilation step.
+// Real java function apps would have this as .java extension.
+
```

Comment:
We have (in the past) used these as direct samples for customers to reference. It's unfortunate if this is the only way we can do this. Is there not a way to ignore files or folders?

Reply:
Our options are:

1. Rename the file (done in this PR)
2. Configure CodeQL to ignore Java. But reporting will _still_ expect a Java run, and so we will also need to file for an exception.
3. Add a Java build task in a separate stage to satisfy CodeQL.

Given the simplicity and that we have samples in other locations, I went for option 1. If this is an issue we can explorer the other options, but maybe at a later date?